firmware/smbios/smbios.ksy

```diff
@@ -0,0 +1,37 @@
+meta:
+  id: smbios
+  endian: le
+  imports:
+    - smbios_bios_info
+    - smbios_system_info
+    - smbios_processor_info
+    - smbios_portable_battery
+    - smbios_skipper
+    - smbios_strings
```

Comment:
This is obviously the main entry point to this project. It really warrants at least some documentation, explaining:

* What SMBIOS is
* How it's used
* How to get SMBIOS file dumps / access it in memory, etc